pkgs/top-level/python-packages.nix

```diff
@@ -7748,6 +7748,26 @@ in {
     };
   });
 
+
+  htmlmin = buildPythonPackage rec {
```

Comment:
Please put this expression into a separate file in ./pkgs/development/python-modules